frontend/src/store.ts

```diff
@@ -5,6 +5,7 @@ import AuthDto from '@/models/user/AuthDto';
 import Course from '@/models/user/Course';
 import AuthUser from '@/models/user/AuthUser';
 import ExternalUser from '@/models/user/ExternalUser';
+import enumerate = Reflect.enumerate;
```

Comment:
why it is necessary?

Reply:
This import was not used, I don't know why it was there.